### PR TITLE
instructions update + console error fix with missing assets

### PR DESCRIPTION
**Pull request by gajen0981**
1. Updates to instructions 
2. Fixes local assets references and eliminate the console errors
3. Fixes  some of the formatting to standard formatting + ESLint

**Before**
image

**After**
image

### CODE REVIEW

**Comment by jmandel** [comment]
Thanks for submitting these! Can we separate this out into the minimal changes required to fix missing assets, vs broader formatting changes that touch hundreds of lines of code? (Ideally for the formatting, this is done with automated rules -- can you clarify how you applied these fixes, so we avoid thrashing on them in future updates?)

File: README.md [hunk 1]
```diff
@@ -2,30 +2,29 @@
 
 ### Mobile Wallet demo at https://c19.cards
 
-* Click "Connect to Lab and get tested" to retrieve a VC
-* Review details (URLs, DIDs, keys, etc) in Dev Tools
-* Menu > "Scan QR Code" to share your VC with a verifier (e.g., the Verifier demo below)
+- Click "Connect to Lab and get tested" to retrieve a VC
```
Comment (jmandel) [inline README.md:5]: Is there a reason to replace `*` with `-` here?

Comment (gajen0981) [reply README.md:5]: thanks Josh -- 
It's one of the Markdown linting rules. 

image

**Comment by gajen0981** [comment]
> Thanks for submitting these! Can we separate this out into the minimal changes required to fix missing assets, vs broader formatting changes that touch hundreds of lines of code? (Ideally for the formatting, this is done with automated rules -- can you clarify how you applied these fixes, so we avoid thrashing on them in future updates?)

Yes, makes sense to separate the core fixes first so its minimized changes. 

Also can you make sure you use standard ESLint + Prettifier.
image

**Comment by gajen0981** [comment]
I will close this PR and create a minimal one

File: README.md (hunk 1 above)
Comment (jmandel) [reply README.md:5]: Thanks -- the rules are just "don't mix and match" -- they don't prefer `*` to `-` if I'm reading right. But I merged your PR anyway ;-)

**Pull request closed without merging**